src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreWorker.cs

```diff
@@ -393,8 +408,15 @@ public async Task<bool> RestoreAsync(SolutionRestoreRequest request, Cancellatio
                     using (var restoreOperation = new BackgroundRestoreOperation())
                     {
                         await PromoteTaskToActiveAsync(restoreOperation, token);
-
-                        var result = await ProcessRestoreRequestAsync(restoreOperation, request, token);
+                        var restoreTrackingData = GetRestoreTrackingData(
```

Comment:
It looks this one `restoreTrackingData` assigned value but never used. Is it intentional?

Reply:
fixed.